packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -302,13 +325,19 @@ export default decorate([
     },
   }),
   injectState,
-  ({ state, effects, remotes = {}, intl: { formatMessage } }) => (
+  ({
+    state,
+    effects,
+    remotes = {},
+    remotesInfo = {},
+    intl: { formatMessage },
+  }) => (
     <div>
       {!isEmpty(remotes.file) && (
         <div>
           <h2>{_('remoteTypeLocal')}</h2>
           <SortedTable
-            collection={remotes.file}
+            collection={_remoteWithInfo(remotes.file, remotesInfo)}
```

Comment:
Do not create new values (objects, functions, arrays) in render, use a [computed](https://github.com/JsCommunity/reaclette#computed--string-compute-) instead.

Reply:
I just updated a version with computed but I have a problem. The data does not refresh. I have to change the page and come back to have data. And if I remove `if (remotesInfo)`, I have data right away but also an error as soon as you get to the page because the data of `remotesInfo` are `undefined` during the first pass.